controller/build/build.go

```diff
@@ -99,38 +99,18 @@ func RunBuild(ctx context.Context, dockerCli command.Cli, in controllerapi.Build
 		return nil, nil, err
 	}
 	if in.ExportPush {
-		if in.ExportLoad {
-			return nil, nil, errors.Errorf("push and load may not be set together at the moment")
-		}
-		if len(outputs) == 0 {
-			outputs = []client.ExportEntry{{
-				Type: "image",
-				Attrs: map[string]string{
-					"push": "true",
-				},
-			}}
-		} else {
-			switch outputs[0].Type {
-			case "image":
-				outputs[0].Attrs["push"] = "true"
-			default:
-				return nil, nil, errors.Errorf("push and %q output can't be used together", outputs[0].Type)
-			}
-		}
+		outputs = append(outputs, client.ExportEntry{
```

Comment:
I think we still need to do specific behavior based on the current value. If the existing value has image exports then these should be set to be pushed.

Additionally, if we don't do this then it is now an error for older daemons that just defined the same output multiple times.